src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -203,6 +209,29 @@ class NotebookFindInput extends FindInput {
 		}));
 		this._actionbar.push(this._filtersAction, { icon: true, label: false });
 	}
+}
+
+export class NotebookFindInput extends FindInput {
+	private _findFilter: NotebookFindInputFilter;
+	private _filterChecked: boolean = false;
```

Comment:
looking at this again, it makes sense for `_filterChecked` to be on `NotebookFindInput`, since it's the main setter/getter of that field. The `NotebookFindInputFilter` just uses it when calls `applyStyles`, which is from the `NotebookFindInput` anyways.